client/src/QuestionsAndAnswerComponents/QuestionAnswerList.jsx

```diff
@@ -12,8 +12,16 @@ class QuestionAnswerList extends Component {
       <Container>
         <Card>
           <Card.Body>
-            <Card.Title >Questions and Answers</Card.Title>
+            <p>QUESTIONS {'&'} ANSWERS</p>
```

Comment:
I like how you put "&" in as a  string instead of using those weird "%="+ html characters